core/src/main/resources/jenkins/split-plugin-cycles.txt

```diff
@@ -10,3 +10,6 @@ script-security command-launcher
 script-security matrix-auth
 script-security matrix-project
 script-security windows-slaves
+script-security jaxb
```

Comment:
Would it apply to any detached plugin?

Reply:
I'm quite unclear about why this seemed to surface only for JAXB TBH. Through some local testing, it seems like it can be random (?!?) because I wasn't able to reproduce the `StackOverflowError` locally, when this was visible in CI (before I added these lines to `split-plugin-cycles.txt`)

Reply:
It seems to be YOLO in the current state. I suggest adding break cycles for all previously detached plugins

Reply:
Still needs discussion

Reply:
Done @oleg-nenashev